### DISCUSSION TITLE
Fira Code Light without ligatures

### DISCUSSION THREAD

**Discussion by skytwosea** [tonsky/FiraCode#1554] [Q&A]
I desperately want the Fira Code Light weight with NO ligatures. Is this available, has anyone else accomplished this? If not, I'm more than willing to try to modify the build or even fork the repo.  I could use a bit of advice on walking that path. Thanks

**Comment by cormullion** [comment]
You can run a font through

https://mutsuntsai.github.io/fontfreeze/

to add/remove any features you want.

**Comment by andy0130tw** [comment]
I have attempted removing all ligatures by hand, but it has not been updated for a while: https://github.com/qbane/FiraCodeNL

**Comment by tonsky** [comment]
Take Fira Mono?
